### ISSUE TITLE
Firefox: JavaScript when creating a link with the whole paragraph

### ISSUE THREAD

**Issue by dpobel** [liferay/alloy-editor#645]
This happens in Firefox (not in Chrome, not tested in IE/Edge) and can be reproduced with http://alloyeditor.com/demo/ 

# Steps to reproduce

* add a paragraph with some words in AlloyEditor
* select the complete paragraph
* click the "Link" button, put whatever URL and validate

The console displays the following message:
```
IndexSizeError: Index or size is negative or greater than the allowed amount  alloy-editor-all-min.js:447
TypeError: t is undefined  alloy-editor-all-min.js:863:14015
```

**Comment by antoniopol06** [comment]
Hi @dpobel !

Thanks to report that.

Could you verify if this issue is the same than [this](https://github.com/liferay/alloy-editor/issues/620) ?

This issue was fixed on master and it will be able into the next version of AlloyEditor 1.3.0.

Let me know if the issue appears on master branch to work on that.

Thanks!!!

**Comment by jbalsas** [comment]
Closing as inactive. @dpobel, please reopen if this isn't fixed in the released `1.3.0`.

Thanks!
